apps/client/src/features/viewers/lower-thirds/LowerThird.tsx

```diff
@@ -0,0 +1,216 @@
+import { useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { Message, OntimeEvent } from 'ontime-types';
+
+import NavigationMenu from '../../../common/components/navigation-menu/NavigationMenu';
+import { LOWER_THIRD_OPTIONS } from '../../../common/components/view-params-editor/constants';
+import ViewParamsEditor from '../../../common/components/view-params-editor/ViewParamsEditor';
+import { cx } from '../../../common/utils/styleUtils';
+
+import './LowerThrid.scss';
+
+//TODO: add none option
```

Comment:
what is a none option?

Reply:
If the user only wants one line

Reply:
changed this so that no text from the source means that the line won't be shown